packages/flutter/lib/src/services/flutter_version.dart

```diff
@@ -33,15 +35,17 @@ abstract final class FlutterVersion {
           ? String.fromEnvironment('FLUTTER_FRAMEWORK_REVISION')
           : null;
 
-  /// The Flutter engine revision.
+  /// The Flutter engine revision, as a (short) Git commit ID.
   static const String? engineRevision =
       bool.hasEnvironment('FLUTTER_ENGINE_REVISION')
           ? String.fromEnvironment('FLUTTER_ENGINE_REVISION')
           : null;
 
-  // This is included since [Platform.version](https://api.dart.dev/stable/dart-io/Platform/version.html)
-  // is not included on web platforms.
```

Comment:
I think we should keep this comment, right?

Reply:
I was thinking it's no longer needed due to the addition of the "On `dart:io` platforms it is more reliable to use [Platform.version]." section in the docs, but I'll revert the removal.

Reply:
Ah I see, sounds good either way.